doc/related_projects.rst

```diff
@@ -207,6 +207,23 @@ Other packages useful for data analysis and machine learning.
 - `Deep Learning <http://deeplearning.net/software_links/>`_ A curated list of deep learning
   software libraries.
 
+Recommendation Engine packages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- `Faster Implicit Matrix Factorization <http://www.benfrederickson.com/fast-implicit-matrix-factorization/>`_ This is a pretty good implementation of the ALS algorithm for implicit matrix factorization.
```

Comment:
I'd avoid giving opinions, and perhaps make it as matter-of-fact/unbiased as possible?